ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -423,6 +474,8 @@ newtype LedgerBackingStore m l = LedgerBackingStore
     )
   deriving (NoThunks)
 
+type LedgerBackingStore' m blk = LedgerBackingStore m (ExtLedgerState blk)
```

Comment:
In contrast to the other synonyms, I do not object to this one.  But that's because these modules had already for awhile now had the `LedgerDB'` synonym, so this one is a continuation of an existing pattern (a pattern which I probably would have voted against were I around when it was introduced).

Reply:
Right, this one follows the pattern exactly as you say. I wouldn't object to keeping it. I'm keeping it for now (we can always remove it later)